Classes/Service/RegistrationProcessor.php

```diff
@@ -169,6 +169,10 @@ public function createAdditionalPersons(Registration $registration, int $storage
             $person->setName($name);
             $email = (string)($singleUserData['email'] ?? '');
             $person->setEmail($email);
+            // Consider additional attendees as having just logged in, so set their last login to now
+            if (\method_exists($person, 'setLastLogin')) {
+                $person->setLastLogin(new \DateTimeImmutable());
```

Comment:
You‘ll need to use the current date/time from the context API instead:

https://docs.typo3.org/m/typo3/reference-coreapi/12.4/en-us/ApiOverview/Context/Index.html#date-time-aspect